taskt/UI/Forms/UIForm.cs

```diff
@@ -12,90 +12,88 @@
+                    return (Height / 2);
                 else
-                {
-                    return backgroundChangeIndex;
-                }
+                    return _backgroundChangeIndex;
             }
             set
             {
-                this.backgroundChangeIndex = value;
-                this.Invalidate();
+                _backgroundChangeIndex = value;
+                Invalidate();
             }
         }
 
-        protected override void OnPaintBackground(System.Windows.Forms.PaintEventArgs e)
+        private void InitializeComponent()
+        {
+            ComponentResourceManager resources = new ComponentResourceManager(typeof(UIForm));
+            SuspendLayout();
+
+            // UIForm
+            ClientSize = new Size(284, 261);
+            Icon = ((Icon)(resources.GetObject("$Icon")));
+            Name = "UIForm";
+            Load += new EventHandler(UIForm_Load);
+            ResumeLayout(false);
+
+        }
+        private void UIForm_Load(object sender, EventArgs e)
+        {
+
+        }
+
+        protected override void OnPaintBackground(PaintEventArgs e)
         {
             base.OnPaintBackground(e);
 
-            //using (var brush = new System.Drawing.Drawing2D.LinearGradientBrush
-            //      (new Rectangle(0, 0, this.Width, BackgroundChangeIndex), Color.SteelBlue, Color.FromArgb(106, 160, 204), System.Drawing.Drawing2D.LinearGradientMode.Vertical))
+            //using (var brush = new LinearGradientBrush
+            //      (new Rectangle(0, 0, Width, BackgroundChangeIndex), Color.SteelBlue, Color.FromArgb(106, 160, 204),
+            //      LinearGradientMode.Vertical))
             //{
-            //    e.Graphics.FillRectangle(brush, 0, 0, this.Width, BackgroundChangeIndex);
+            //    e.Graphics.FillRectangle(brush, 0, 0, Width, BackgroundChangeIndex);
             //}
 
-            //using (var brush = new System.Drawing.Drawing2D.LinearGradientBrush
+            //using (var brush = new LinearGradientBrush
 
-            //(new Rectangle(0, BackgroundChangeIndex, this.Width, this.Height), Color.FromArgb(255, 214, 88), Color.Orange, System.Drawing.Drawing2D.LinearGradientMode.Vertical))
+            //(new Rectangle(0, BackgroundChangeIndex, Width, Height), Color.FromArgb(255, 214, 88), Color.Orange,
+            //LinearGradientMode.Vertical))
             //{
-            //    e.Graphics.FillRectangle(brush, 0, BackgroundChangeIndex, this.Width, this.Height);
+            //    e.Graphics.FillRectangle(brush, 0, BackgroundChangeIndex, Width, Height);
             //}
+
             var topColor = Color.FromArgb(49, 49, 49);
-            using (var brush = new System.Drawing.Drawing2D.LinearGradientBrush
-                  (new Rectangle(0, 0, this.Width, BackgroundChangeIndex), topColor, topColor, System.Drawing.Drawing2D.LinearGradientMode.Vertical))
+            using (var brush = new LinearGradientBrush
```

Comment:
This is another example where we should move complexity out of the parameters of a method and into the lines above (in this case the constructor for LinearGradientBrush also has a call to a constructor for a Rectangle). Maybe define some variable brushRectangle and pass that in instead. 

Also (and I know the code was like this when you got it) we should try and keep the opening paranthesis of the LinearGradientBrush constructor args on the same line as the class declaration, like so:
```
using (var brush = new LinearGradientBrush(
    Arg1,
    Arg2,
    ...etc.
    )
)
```
Or have the opening parenthesis be its own line so there is a clear break when the constructor args begin, like so:
```
using (var brush = new LinearGradientBrush
    (
    Arg1,
    Arg2,
    ...etc.
    )
)
```
(These also look a lot better when you have good spacing which I do not here, hah)

Reply:
I've changed it but it still looks pretty bad regardless